app/models/yellow_ribbon_program.rb

```diff
@@ -5,6 +5,9 @@ class YellowRibbonProgram < ApplicationRecord
 
   delegate :country, :insturl, to: :institution
   delegate :institution, to: :institution, prefix: :name_of
+  delegate :latitude, :longitude, :ungeocodable, to: :institution
+  delegate :online_all, :online_only, :distance_learning, :correspondence, to: :institution
+  delegate :student_veteran, :student_veteran_link, :student_vet_grp_ipeds, to: :institution
```

Comment:
Maybe these could be added to line 6, just tacking on the attributes that are delegated to institution?

Reply:
> Maybe these additional lines could be added to line 6, just tacking on the attributes that are delegated to institution?

I considered this, and it would have made that line excessively long, so I went with adding the new items as separate statements grouped by usage.

Alternately, everything could be combined into one statement alphabetically with line breaks after each item.  This would match the style used in the Serializer class.

Reply:
> Are you guys using a JSON schema for this and if so, has that been updated as needed?

The yellow_ribbon_program schema used by unit test code has been updated in the latest commit.